test/tools/router_check/json/tool_config_schemas.cc

```diff
@@ -60,6 +60,19 @@ const std::string& Json::ToolSchema::routerCheckSchema() {
                  "required": ["field", "value"],
                  "maxProperties": 2
                 }
+              },
+              "custom_header_fields": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "field": {"type": "string"},
+                    "value": {"type": "string"}
+                  },
+                 "additionalProperties": false,
+                 "required": ["field", "value"],
+                 "maxProperties": 2
```

Comment:
why do we need this restriction? it already has 2 required properties.

Reply:
I am trying to mimic the "header_fields" property definition right above it on line 51. To my understanding and from the JSON schema used by rapid json ([here](https://spacetelescope.github.io/understanding-json-schema/UnderstandingJSONSchema.pdf)), it looks like the above is enforcing that the actual JSON doc should have both the "field" and "value" property.

Reply:
required: ["field", "value"] is enough to force presence of those two fields. Yeah, I see that in a bunch of other places. It's ok, it's just redundant.